parmed/modeller/residue.py

```diff
@@ -84,6 +88,7 @@ def __init__(self, name=''):
         self.first_patch = None
         self.last_patch = None
         self.groups = []
+        self.omm_overloadLevel = 0
```

Comment:
Let's just keep this named `overload_level`, or perhaps `priority` would be a better name for it?